types/dom-chromium-ai/index.d.ts

```diff
@@ -98,7 +98,7 @@ interface AILanguageModelSystemPrompt extends AILanguageModelInitialPrompt {
 
 type AILanguageModelPromptInput = string | AILanguageModelPrompt | AILanguageModelPrompt[];
 
-interface AILanguageModel extends EventTarget {
+interface AILanguageModel {
```

Comment:
It seems like `AILanguageModel ` still extends `EventTarget` in Chromium: https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/modules/ai/ai_language_model.idl;l=20;drc=0ec3372000136102dc63cd3d7ee680c42ccc84c0

Reply:
Undoing the change then. Thanks!